.drone.star

```diff
@@ -1100,7 +1100,7 @@ def smokeTests(ctx):
             "OCIS": "true",
         },
         "commands": [
-            "sleep 10 && yarn test:smoke:experimental tests/smoke/features/",
+            "sleep 10 && yarn test:e2e:cucumber tests/e2e/cucumber/",
```

Comment:
Why the `cucumber` part in the yarn cmd? I'd like it to be short&easy to type 😬

Reply:
cucumber is the bdd framework which drives our playwright tests, the tests are implemented abstract.
This means were able to have multiple e2e flavoured tests. For example one which uses playwright testing library. 

The idea is to have different wrappers for different needs, gherkin is nice for journeys written by product owner, but i expect that we try out to have also some integration like tests in place sooner or later. this is why we mention cucumber in the script